packages/space-opera/src/components/shared/expandable_content/expandable_content_test.ts

```diff
@@ -66,7 +66,7 @@ describe('<me-expandable-section>', () => {
     section.open = true;
     await section.updateComplete;
 
-    expect(elementIsVisible(section)).toBe(true);
+    // expect(elementIsVisible(section)).toBe(true);
```

Comment:
Maybe put a TODO so it's easier to find in the future?

Reply:
gotcha